src/server/webpack-plugin/server.js

```diff
@@ -10,7 +10,8 @@ export default class VueSSRServerPlugin {
   apply (compiler) {
     validate(compiler)
 
-    onEmit(compiler, 'vue-server-plugin', (compilation, cb) => {
+    const stage = 'PROCESS_ASSETS_STAGE_OPTIMIZE_TRANSFER'
```

Comment:
Why not just put the stage name in the `onEmit` function? It's not used anywhere else

Reply:
@sodatea client plugin and server plugin are applied in different stage, so I think it's better to specify the stage name here.